samples/cpp/CMakeLists.txt

```
@@ -59,7 +59,11 @@ endif()
 
 if(OpenVINO_SOURCE_DIR)
     # in case if samples are built from OV repo
-    set(OV_MAIN_SAMPLES_DIR "${OpenVINO_SOURCE_DIR}")
+    if (DEFINED OUTPUT_ROOT)
+        set(OV_MAIN_SAMPLES_DIR ${OUTPUT_ROOT})
+    else()
+        set(OV_MAIN_SAMPLES_DIR ${OV_MAIN_SAMPLES_DIR})
```

This is probably a typo
```suggestion
        set(OV_MAIN_SAMPLES_DIR ${OpenVINO_SOURCE_DIR})
```

---

yeah thank you!